README.md

```diff
@@ -2,25 +2,50 @@
 
 ## Goal
 
-Provide a simple web page that allows a user to page through a collection of active user records.
-
-### Assumptions
-* Technology choice is whatever platform you're comfortable in, but try to incoorporate Java into the solution.
-* No CSS styling necessary, seriously!
-* Users per page 5
-* Select 3-4 fields from the user's details to display per page
-* Dynamic paging controls (i.e. if no previous page is available, no previous link is presented)
-* Scalable in terms of not holding the entire user collection in memory, let the API do the work for you.
+In order to increase the efficiency of a client using the FitPay API, we need a new composite
+API resource created that will decrease the number of network calls being made.
+
+### Technology Assumptions
+* Technology choice is whatever platform you're comfortable in
+⋅⋅* If you chose a java based solution, a very basic spring-boot application is included.
 * API credentials and/or bearer tokens can be hard coded with values obtained out-of-band
 * You're familiar with [curl](https://curl.haxx.se) for testing Restful APIs
+* Your solution can ideally be launched with a single command, if not instructions MUST be provided so we can test your new API
+⋅⋅* Example `mvn spring-boot:run`
+
+### Overview
+
+API clients typically need three things when presenting an experience to an end consumer.  The `user`, `devices`, and `creditCards` API resources provided by the FitPay API.  This requires 3 separate network calls to obtain.  The requirement is to create a new API resource called `compositeUser` that allows for all these resources to be returning in a single network call.
```

Comment:
isn't it users not user?

Reply:
"Technically" no, we're asking for a single API resource which is a single composite user and not for a collection of users.  Subtle.

Reply:
I mean our api resrouce it is not /user it is /users 

https://api.qa.fitpay.ninja/users/:userId